client/resource_table_caching.go

```diff
@@ -68,6 +68,11 @@ func ResourceTableCaching() *schema.Resource {
 				Elem:          cronScheduleSchema(),
 				ConflictsWith: []string{"daily_schedule"},
 			},
+			"principal": {
```

Comment:
nit pick. Indentation is off

Reply:
You can run `go fmt client/*` to get these right.